### PR TITLE
Add frontend SV

### PR DESCRIPTION
**Pull request by aapeliv**
**Web frontend checklist**
- [ ] Formatted my code with `yarn format`
- [ ] There are no warnings from `yarn lint --fix`
- [ ] There are no console warnings when running the app
- [ ] Added any new components to storybook
- [ ] Added tests where relevant
- [ ] All tests pass
- [ ] Clicked around my changes running locally and it works
- [ ] Checked Desktop, Mobile and Tablet screen sizes

### CODE REVIEW

**Comment by aapeliv** [comment]
@jesseallhands could you please check the wording around the place and suggest any improvements to clarify?

File: app/web/features/auth/verification/CompleteStrongVerification.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+export default function CompleteStrongVerification() {
+  const { t } = useTranslation(AUTH);
+
+  const router = useRouter();
+  const verificationAttemptToken = stringOrFirstString(
+    router.query.verification_attempt_token
+  );
+
+  console.info("Captured token", verificationAttemptToken);
```
Comment (nexagoai) [inline app/web/features/auth/verification/CompleteStrongVerification.tsx:17]: remove?

Comment (nexagoai) [reply app/web/features/auth/verification/CompleteStrongVerification.tsx:17]: [I'm David xd]

File: app/web/features/auth/locales/en.json [hunk 2]
```diff
@@ -330,8 +330,20 @@
       "enabled_message": "You <1>are currently</1> verified with Strong Verification.",
       "disabled_message": "You <1>are currently not</1> verified with Strong Verification."
```
Comment (jesseallhands) [inline app/web/features/auth/locales/en.json]:
```suggestion
      "disabled_message": "You <1>are not currently</1> verified with Strong Verification."
```

File: app/web/features/auth/locales/en.json [hunk 3]
```diff
@@ -330,8 +330,20 @@
...
     },
-    "complete_in_console": "You can complete Strong Verification <2>on this page</2>. We are working on improving the flow and implementing it fully in the platform.",
-    "go_to_button": "Go to Strong Verification settings"
+    "start_button": "Start Strong Verification",
+    "begin_button": "Complete Strong Verification with Iris ID",
+    "information_text1": "You will need a biometric passport and an NFC-capable Apple iPhone or Android phone.",
```
Comment (jesseallhands) [inline app/web/features/auth/locales/en.json:335]: A lot of people will not know what a biometric passport is. I didn't know for sure the first time, so I had to look it up. Perhaps we can just link to the wikipedia page? https://en.wikipedia.org/wiki/Biometric_passport

**Review by nabramow — DISMISSED** [review]
Approving as we confirmed on a call that it works to go through to Iris API and you can scan the QR code on the phone.

I'll do the real strong verification once it's merged.

File: app/web/features/auth/locales/en.json [hunk 4]
```diff
@@ -330,8 +330,20 @@
...
+    "information_text2": "By clicking the button below, you will be taken to the Iris ID website to complete the passport verification process.",
+    "information_text3": "You can read more about how we and Iris ID process and store your data in our <2>Terms of Service</2>. As per the Iris ID <5>Privacy policy</5>, they will delete your personal information within 24 hours. You can request for this Strong Verification data (except issuing country, expiry date, and the last three digits of the document number) to be deleted from our servers at any time with the click of a button.",
```
Comment (jesseallhands) [inline app/web/features/auth/locales/en.json]:
```suggestion
    "information_text3": "You can read more about how we and Iris ID process and store your data in our <2>Terms of Service</2>. As per the Iris ID <5>Privacy policy</5>, they will delete your personal information within 24 hours. You can request that this Strong Verification data (except issuing country, expiry date, and the last three digits of the document number) be deleted from our servers at any time with the click of a button.",
```

File: app/web/features/auth/locales/en.json [hunk 5]
```diff
@@ -330,8 +330,20 @@
...
+    "complete_title": "We are processing your Strong Verification attempt",
+    "complete_message1": "Your passport is being validated. If it matches your gender and date of birth, you will soon receive an email within approximately 15 minutes.",
```
Comment (jesseallhands) [inline app/web/features/auth/locales/en.json]: The 15 minute wait time seems really long. Why does it take so long to process?

File: app/web/features/auth/locales/en.json [hunk 6]
```diff
@@ -330,8 +330,20 @@
...
+    "complete_message2": "Strong Verification expires when your passport expires, and you will need to complete verification again on a new passport.",
```
Comment (jesseallhands) [inline app/web/features/auth/locales/en.json]:
```suggestion
    "complete_message2": "Strong Verification expires when your passport expires, and you will need to complete verification again with the passport that replaces the expired one.",
```

**Review by jesseallhands — CHANGES_REQUESTED** [review]

File: app/web/features/auth/locales/en.json (hunk 5 above)
Comment (aapeliv) [reply app/web/features/auth/locales/en.json]: I actually realized I fixed that, changed to "a few minutes"

**Review by jesseallhands — APPROVED** [review]

**Pull request merged**